Sources/Controllers/Map/OAMapViewController.h

```diff
@@ -186,6 +186,7 @@
 
 - (void) onApplicationDestroyed;
 
-- (void) updateLayerProviderAlpha;
+- (void) updateRasterLayerProviderAlpha:(float)alpha;
+- (void) updateSymbolsLayerProviderAlpha;
```

Comment:
These methods should be private. Use observers inside OAMapViewController.h and call updates accordingly.